ui/page/dcrdex/dcrdex_page.go

```diff
@@ -34,6 +39,9 @@ func NewDEXPage(l *load.Load) *DEXPage {
 		MasterPage:        app.NewMasterPage(DCRDEXID),
 		openTradeMainPage: l.Theme.NewClickable(false),
 	}
+
+	dp.initSplashPageWidgets()
+	dp.navigateToSettingsBtn = dp.Theme.Button(values.String(values.StrStartTrading))
```

Comment:
Oh, I see the text on the button now. Please rename `navigateToSettingsBtn` to `finalizeOnboardingBtn`. To confirm, this is the button that is used to exit the onboarding view, correct?